setup.py

```diff
@@ -46,8 +47,9 @@
     version=djangocms_versioning_filer.__version__,
     description=djangocms_versioning_filer.__doc__,
     dependency_links=[
-        "http://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
-        "http://github.com/divio/djangocms-versioning/tarball/master#egg=djangocms-versioning-0.0.24",
+        "https://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
+        "https://github.com/divio/djangocms-moderation/tarball/release/1.0.x#egg=djangocms-moderation-1.0.22",
```

Comment:
What's the reason for Moderation here? This package would ideally be seperate, I'm guessing that you've found tests or code that makes this package dependant or compatible somehow.

Reply:
If you take a look at tox.ini you'll see that it's testing some stuff with moderation as well. You can also look at tests. Some tests are skipped if moderation is not enabled.